cmd/clusterctl/examples/openstack/provider-component/user-data/ubuntu/templates/worker-user-data.sh

```diff
@@ -79,6 +79,8 @@ CLUSTER_DNS_SERVER=$(prips ${SERVICE_CIDR} | head -n 11 | tail -n 1)
 
 # Write the cloud.conf so that the kubelet can use it.
 echo $OPENSTACK_CLOUD_PROVIDER_CONF | base64 -d > /etc/kubernetes/cloud.conf
+mkdir /etc/certs
+echo $OPENSTACK_CLOUD_CACERT_CONFIG | base64 -d > /etc/certs/cacert
```

Comment:
Just curious 
this is only for ubuntu, do we have a centos version? I didn't see it ...

Reply:
@jichenjc Thanks pointing out. I added in https://github.com/kubernetes-sigs/cluster-api-provider-openstack/commit/b06fd90d5eaff2c379e3322bf8bcbc5e3a60f3bc

Reply:
Thanks added this for centos.